src/BeamAdapter/component/constraint/AdaptiveBeamLengthConstraint.inl

```diff
@@ -50,9 +50,9 @@ public:
     AdaptiveBeamLengthConstraintResolution(double* initF=NULL, bool* active=NULL) : ConstraintResolution(1) ,m_initF(initF), m_active(active)
     {
     }
-    virtual void init(int line, double** w, double* force);
-    virtual void resolution(int line, double** w, double* d, double* force);
-    virtual void store(int line, double* force, bool convergence);
+    void init(int line, double** w, double* force) override;
+    void resolution(int line, double** w, double* d, double* force) override;
```

Comment:
This one does not override anything. And I bet it is a bug. I think it should override `resolution(int line, double** w, double* d, double* force, double* dfree)`. Otherwise, it is never called.